### PR TITLE
Created project base and added first controller

### PR DESCRIPTION
**Pull request by arinaldi118**
## Summary
- Completed the project setup
- Added first controller
- Added first view

## Trello Card
[https://trello.com/c/oELYakK8/17-crear-un-controller-y-una-vista-de-bienvenida](url)

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,7 @@
+spring.datasource.url=jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}
+spring.datasource.username=${DB_USERNAME}
+spring.datasource.password=${DB_PASSWORD}
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto = update
+
+server.port = ${PORT}
```
Comment (cramallo) [inline src/main/resources/application.properties]: add eof

Comment (arinaldi118) [reply src/main/resources/application.properties]: :thinking: eof == a empty line??

Comment (cramallo) [reply src/main/resources/application.properties]: yes, it means "end of file", it's a good practice

Comment (arinaldi118) [reply src/main/resources/application.properties]: :+1:

**Review by cramallo — APPROVED** [review]

**Pull request merged**